api-schema/src/main/java/org/stellar/anchor/api/rpc/RpcRequest.java

```diff
@@ -0,0 +1,13 @@
+package org.stellar.anchor.api.rpc;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class RpcRequest {
+  private Object id;
```

Comment:
Pretty sure it's just String

> id
    An identifier established by the Client that MUST contain a String, Number, or NULL value if included. If it is not included it is assumed to be a notification. The value SHOULD normally not be Null [[1]](https://www.jsonrpc.org/specification#id1) and Numbers SHOULD NOT contain fractional parts [[2]](https://www.jsonrpc.org/specification#id2)

(from specs)

Reply:
Request may contain `"id": "1"` or `"id": 1`.
String will handle both cases and convert to String with value "1"
Response will contain String as well, even if request contained Number. But it's shoun't be a problem to convert it back to Number.

Reply:
Oh yeah you are right gson treats it differently. Let's keep it like this then